src/mappers/manfred-to-europass.mapper.ts

```diff
@@ -0,0 +1,52 @@
+import { SkillsPassport } from "@/models/europass"
+import { ManfredAwesomicCV } from "@/models/manfred"
+
+
+export const manfredToEuropass = (manfred: ManfredAwesomicCV): SkillsPassport => {
+    const europass: SkillsPassport = {
+        documentInfo: {
+            documentType: 'ECV',
+            creationDate: new Date(Date.now()),
+            lastUpdateDate: new Date(Date.now()),
+            xsdVersion: 'V3.0',
+            generator: 'EWA',
+            comment: 'Europass CV'
+        },
```

Comment:
Crate a separate function for this, this should be:

```typescript
    const europass: SkillsPassport = {
        documentInfo: generateEuropassDocumentInfo(),
```

```typescript
const generateEuropassDocumentInfo = () :DocumentInfo => {
  return {
            documentType: 'ECV',
            creationDate: new Date(Date.now()),
            lastUpdateDate: new Date(Date.now()),
            xsdVersion: 'V3.0',
            generator: 'EWA',
            comment: 'Europass CV'
  }
}
```

Reply:
done